var/spack/repos/builtin/packages/process-in-process/package.py

```diff
@@ -27,157 +37,167 @@ class ProcessInProcess(Package):
     conflicts("%gcc@:7", when="os=rhel8")
     conflicts("%gcc@9:", when="os=rhel8")
 
+    # packages required for building PiP-glibc
+    #    currently none
+
     # packages required for building PiP-gdb
-    depends_on("texinfo", type="build")
-    depends_on("systemtap")
+    if enable_pip_gdb:
```

Comment:
Can't we turn this into a variant? I think of something similar to:
```suggestion
    with when('+gdb'):
```